ext/gitsh/src/line_editor.c

```diff
@@ -1544,7 +1544,7 @@ Init_line_editor_native(void)
     VALUE history, fcomp, ucomp, version;
 
     /* Allow conditional parsing of the ~/.inputrc file. */
-    rl_readline_name = (char *)"Ruby";
+    rl_readline_name = (char *)"gitsh";
```

Comment:
Is the cast to `char *` needed?

Reply:
Probably not, but I've tried to keep the changes from the upstream Ruby `Readline` module to a minimum.